byu-calendar/byu-calendar-common.scss

```diff
@@ -29,4 +29,6 @@
 .fullpage-date-text {
     padding-left: 10px;
     font-family: "Sentinel A", "Sentinel B", "Vitesse A", "Vitesse B", sans-serif;
-}
\ No newline at end of file
+}
+
+@import '../byu-calendar-feature-column/byu-calendar-feature-column-extras';
```

Comment:
This was added because there were some necessary styles that needed to get added to the overall feature wrapper. The file byu-calendar-feature-column.scss couldn't style the overall feature wrapper. This file has like three rules in it that just deal with the flexbox styling of the component.

Reply:
I also noticed that there were a few rules in the byu-calendar.scss file that the image rows were using for the date wrappers. I kept the rules for the feature component in a separate file just to make sure I remembered what the rules were for. I can put the rules just directly in there instead of having the stylesheet being imported.